style/roads.mss

```diff
@@ -713,10 +711,9 @@
       }
     }
 
-    [feature = 'highway_cycleway'],
-    [feature = 'highway_path'][bicycle = 'designated'] {
```

Comment:
No, this was correct, since `highway=path` only exists for "real" paths, so there are no alternative formulations of cycleways / bridleways needed in MSS.